include/revng/PipelineC/Prototypes.h

```diff
@@ -356,6 +363,12 @@ const rp_kind *rp_analysis_get_argument_acceptable_kind(rp_analysis *analysis,
                                                         int argument_index,
                                                         int kind_index);
 
+/**
+ * Serialize a single pipeline step to the specified folder
+ *  \return 0 if a error happened, 1 otherwise.
+ */
+bool rp_step_save(rp_step *step, const char *path);
```

Comment:
Mainly, to allow selectively saving a single step (e.g. `begin`) to a separate directory.
This is used with the FIFO functionality to avoid serializing *every* step multiple times, which is wasteful.